app/controllers/TaxEnrolmentCallbackController.scala

```diff
@@ -32,7 +32,7 @@ class TaxEnrolmentCallbackController @Inject()(
                                                 cc: ControllerComponents
                                                ) extends BackendController(cc) with Logging {
 
-  def subscriptionCallback() = Action.async(parse.json) {
+  def subscriptionCallback(trn: String) = Action.async(parse.json) {
```

Comment:
Could we rename the method to me taxableSubscriptionCallback?

Reply:
Maybe also include that it was taxable in the log as well?

Reply:
Done